.changeset/late-boats-report.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
```

Comment:
Lets make it minor? It might break snapshots for some folks

Reply:
```suggestion
"@razorpay/blade": minor
```

Reply:
You are so lazy man 😆